wiki/release/release-process.md

```diff
@@ -382,13 +391,13 @@ This is alternative to A. when the user has access to apachepulsar org in hub.do
 
 4. Once it is merged, the chart will be automatically released to Github releases at https://github.com/apache/pulsar-helm-chart and updated to https://pulsar.apache.org/charts.
 
-## 12. Publish Python Clients
+## 11. Publish Python Clients
```

Comment:
It's better to swap step 11 and 12.

Reply:
The release scripts of the Python client and the C++ client are independent, though the Python client reuses much C++ client code. I think the order of them is not an issue.

Reply:
I mean swap [11. Publish Python Clients] and [12. Publish MacOS libpulsar package]
So that [13. Update Python Client docs] goes right after "Publish Python Clients"